modules/cluster-logging-deploy-cli.adoc

```diff
@@ -308,35 +315,42 @@ spec:
   managementState: "Managed"  <2>
   logStore:
     type: "elasticsearch"  <3>
+    retentionPolicy: <4>
+      application:
+        maxAge: 1d
+      infra:
+        maxAge: 7d
+      audit:
+        maxAge: 7d
     elasticsearch:
-      nodeCount: 3 <4>
+      nodeCount: 3 <5>
       storage:
-        storageClassName: "<storage-class-name>" <5>
+        storageClassName: "<storage-class-name>" <6>
         size: 200G
       redundancyPolicy: "SingleRedundancy"
   visualization:
-    type: "kibana"  <6>
+    type: "kibana"  <7>
     kibana:
       replicas: 1
   curation:
-    type: "curator"  <7>
+    type: "curator"  <8>
     curator:
       schedule: "30 3 * * *"
   collection:
     logs:
-      type: "fluentd"  <8>
+      type: "fluentd"  <9>
       fluentd: {}
 ----
 <1> The name must be `instance`.
-<2> The Cluster Logging management state. In most cases, if you change the Cluster Logging defaults, you must set this to `Unmanaged`.
-However, an unmanaged deployment does not receive updates until Cluster Logging is placed back into the `Managed` state.
+<2> The cluster logging management state. In some cases, if you change the cluster logging defaults, you must set this to `Unmanaged`.
+However, an unmanaged deployment does not receive updates until the cluster logging is placed back into a managed state.
 <3> Settings for configuring Elasticsearch. Using the Custom Resource (CR), you can configure shard replication policy and persistent storage.
-<4> Specify the number of Elasticsearch nodes. See the note that follows this list.
-<5> Enter the name of an existing StorageClass for Elasticsearch storage. For best performance, specify a StorageClass that allocates block storage.
-<6> Settings for configuring Kibana. Using the CR, you can scale Kibana for redundancy and configure the CPU and memory for your Kibana nodes. For more information, see *Configuring Kibana*.
-<7> Settings for configuring Curator. Using the CR, you can set the Curator schedule. For more information, see *Configuring Curator*.
-<8> Settings for configuring Fluentd. Using the CR, you can configure Fluentd CPU and memory limits. For more information, see *Configuring Fluentd*.
-endif::[]
+<4> Specify the time that Elasticsearch should retain each log source. Logs older than the `maxAge` are deleted.
```

Comment:
The code refers to this as "TimeUnit" which is an integer number and time designation (e.g. w, M, h)  I wonder if we should use similar and specify the actual units allowed for clarity